docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     expose:
       - "27017"
   web:
-    build: .
+    image: rbclark/heimdall:latest
```

Comment:
Do we want to push your image name to master?

Reply:
In order to change that we need control of the mitre project on dockerhub, until we have that we can't put the images there.